src/pages/elements/app-icons/production.mdx

```diff
@@ -90,103 +90,29 @@ Stroke app icons feature both color and alpha channel gradients. This technique
 
 ![Exporting for product and web: step 2](/images/3_3-1_ExportingProduct_Step2.png)
 
-### Export stroke app icons
-
-All stroke app icons must be exported as PNG at 2x the apparent icon size. Note that stroke app icons **cannot** be exported as SVG, as SVG does not support this particular styling.
-
-1. Make sure your background layer is turned off.
-2. Select `File > Export > Export for Screens` from the top navigation.
-3. In the `Formats` section, add scales, suffixes, and format to output your icon with proper sizing, naming, and file type. Refer to the table below.
-
-<Row>
-<Column colMd={6} colLg={8}>
-
-![Exporting stroke app icons: step 3](/images/3_3-2_ExportingProduct_Stroke_Step3.png)
-
-</Column>
-</Row>
-
-| Icon size | Scale | Suffix        | Output |
-| --------- | ----- | ------------- | ------ |
-| 16px      | 1x    | AppName-16@2x | 32px   |
-| 20px      | 1.25x | AppName-20@2x | 40px   |
-| 24px      | 1.5x  | AppName-24@2x | 48px   |
-| 32px      | 2x    | AppName-32@2x | 64px   |
-
-4. Click `Export Artboard`. Your files will export to PNG and should reflect the output below.
-
-```
-app-icon-dark-AppName-16@2x.png
-app-icon-dark-AppName-20@2x.png
-app-icon-dark-AppName-24@2x.png
-app-icon-dark-AppName-32@2x.png
-app-icon-light-AppName-16@2x.png
-app-icon-light-AppName-20@2x.png
-app-icon-light-AppName-24@2x.png
-app-icon-light-AppName-32@2x.png
-app-icon-monodark-AppName-16@2x.png
-app-icon-monodark-AppName-20@2x.png
-app-icon-monodark-AppName-24@2x.png
-app-icon-monodark-AppName-32@2x.png
-app-icon-monolight-AppName-16@2x.png
-app-icon-monolight-AppName-20@2x.png
-app-icon-monolight-AppName-24@2x.png
-app-icon-monolight-AppName-32@2x.png
-```
-
-5. Make sure all icons have been exported with proper naming convention.
-
-<Row>
-<Column colMd={6} colLg={8}>
-
-![App icon naming convention (PNG)](/images/3_3-4_ExportingProduct_Stroke_Step5.svg)
-
-</Column>
-</Row>
-
-### Exporting fill and IBM Plex app Icons
-
-Fill app icons and IBM Plex app icons may be exported as SVG. Note that stroke app icons **cannot** be exported as SVG.
-
-1. Make sure your background layer is turned off.
-2. Select `File > Export > Export for Screens` from the top navigation.
-3. In the `Formats` section, add your app’s name in the suffix field.
-
-<Row>
-<Column colMd={6} colLg={8}>
-
-![Exporting fill and IBM Plex app icons: step 3](/images/3_3-5_ExportingProduct_Fill_Step3.png)
-
-</Column>
-</Row>
-
-4. Click `Export Artboard`. Your files will export to SVG and should reflect the output below.
-
-```
-app-icon-dark-AppName.svg
-app-icon-light-AppName.svg
-app-icon-monodark-AppName.svg
-app-icon-monolight-AppName.svg
-```
-
-5. Make sure all icons have been exported with proper naming convention.
-
-<Row>
-<Column colMd={6} colLg={8}>
-
-![App icon naming convention (SVG)](/images/3_3-7_ExportingProduct_Fill_Step5.svg)
-
-</Column>
-</Row>
+3. Make sure your background layer is turned off.
```

Comment:
The list is starting from 1 here instead of 3. 
![image](https://user-images.githubusercontent.com/11670886/67712579-42a28780-f992-11e9-95aa-1da52d1b7d89.png)

Reply:
until this issue is resolved, I can either have the list continue and all images be too small, or have the list break
https://github.com/carbon-design-system/design-language-website/issues/169

Reply:
you should be able to prevent the numbering from resetting with backslash

```suggestion
3\. Make sure your background layer is turned off.
```

and then setting `padding-left: 0` on those lines to fix the alignment